docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -28,19 +28,64 @@ Teleport Agent Installed
 
 ## Step 4/5 Access Server
 
-Select close and Server can be accessed
+Select close and Server can be accessed via SSH
 ![Access Server](../../../img/cloud/online-session-5.png)
 
+Interact with your SSH session
+![Open SSH](../../../img/cloud/online-ssh-5.png)
+
+Exit the session with `exit` and you can replay the session within Session Recordings
+![Replay Session](../../../img/cloud/online-replay-5.png)
+
 ## Step 5/5 Access from Command Line
 
 Install client libraries:
 
 <Tabs>
+  <TabItem label="MacOS">
```

Comment:
This should probably be a partial.

Reply:
@zmb3 moved to partial and updated to use cdns